oidc-controller/api/core/acapy/client.py

```diff
@@ -139,3 +144,305 @@ def oob_create_invitation(
 
         logger.debug("<<< oob_create_invitation")
         return result
+
+    def create_invitation(
```

Comment:
Function above is also creating an oob invitation (same endpoint), though as noted in other comment that one looks to only be called by unreachable code so can decide on removing it.

Reply:
Thanks for catching that. This was the start of what eventually became create_connection_invitation. I can drop this method since it serves no use